pyiron_base/job/jobtype.py

```diff
@@ -127,6 +172,11 @@ def convert_str_to_class(job_class_dict, class_name):
         )
 
 
+def unregistered_jobtype(cls):
+    JobType.un_register_job_type(cls.__name__)
+    return cls
+
+
```

Comment:
```suggestion
```
Like I said above, I'd avoid this function and make the class method itself usable as the decorator.